dcm4che-core/src/main/java/org/dcm4che3/data/BulkData.java

```diff
@@ -333,17 +348,11 @@ public String uriWithoutQuery() {
         return uri.substring(0, uriPathEnd);
     }
 
-    public InputStream openStream() throws IOException {
+    public ImageInputStream openImageInputStream() throws IOException {
```

Comment:
As you removed the openStream() method (replaced it with the openImageInputStream()), does that mean the BulkData now only works for pixeldata, and no longer for other BulkData like overlay data etc.?

Reply:
No, it works for all types of BulkData:  I'm just using IIS internally because it allows positional changes and skipping.  In a subsequent update I restored the openStream() which wraps the underlying IIS.